### PR TITLE
Release 1.21.1

### PR DESCRIPTION
**Pull request by khorne3**

### CODE REVIEW

**Comment by khorne3** [comment]
CC: @deansheather 

I'm not sure if this should be included; wdyt? https://github.com/cdr/m/commit/6da7df035b1705522554b3881688ae26fc68ae5b

File: changelog/1.21.1.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- infra: Add labels to capture workspace image metadata.
+
+### Bug fixes 🐛
+
+- web: Fixed issue so that CVM-enabled workspaces can be built with a minimum of
+  1 GB disk storage.
+- web: Fixed issue where `nodeSelector` and `tolerations` weren't set properly
+  in the UI. parse its version number.
+- web: Fixed issue where UI indicated dev URLs were disabled even when enabled.
+- infra: Fixed issue where JetBrains editors failed if Coder couldn't properly
```
Comment (f0ssel) [inline changelog/1.21.1.md:23]:
```suggestion
- infra: Fixed issue where JetBrains editors failed if Coder couldn't properly read product version manifests.
```

File: changelog/1.21.1.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
```
Comment (f0ssel) [inline changelog/1.21.1.md:24]:
```suggestion
- infra: Fixed issue where process niceness was not being set properly in workspaces.

```

**Review by f0ssel — APPROVED** [review]
Thanks for having this already ready to go, this is great.

**Comment by f0ssel** [comment]
> CC: @deansheather
> 
> I'm not sure if this should be included; wdyt? [cdr/m@6da7df0](https://github.com/cdr/m/commit/6da7df035b1705522554b3881688ae26fc68ae5b)

I added a suggested line for this

**Pull request merged**